### ISSUE TITLE
1D conv can't be padded non-constant

### ISSUE THREAD

**Issue by flozi00** [speechbrain/speechbrain#654]
speechbrain installed from github
the code:

```
myfile = "files/1618507997048-recorded.wav"

from speechbrain.pretrained import SpectralMaskEnhancement

enhance_model = SpectralMaskEnhancement.from_hparams(
    source="speechbrain/mtl-mimic-voicebank",
    savedir="pretrained_models/mtl-mimic-voicebank",
)

result = enhance_model.enhance_file(myfile)

print(result)
```

the log:

```
Traceback (most recent call last):
  File ".\test.py", line 10, in <module>
    result = enhance_model.enhance_file(myfile)
  File "f:\codes\python apps\asr\speechbrain-released\speechbrain\pretrained\interfaces.py", line 805, in enhance_file
    enhanced = self.enhance_batch(noisy)
  File "f:\codes\python apps\asr\speechbrain-released\speechbrain\pretrained\interfaces.py", line 788, in enhance_batch
    mask = self.modules.enhance_model(noisy_features)
  File "C:\Users\flozi\anaconda3\envs\wav2vec\lib\site-packages\torch\nn\modules\module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "f:\codes\python apps\asr\speechbrain-released\speechbrain\lobes\models\transformer\TransformerSE.py", line 88, in forward
    x = self.custom_emb_module(x)
  File "C:\Users\flozi\anaconda3\envs\wav2vec\lib\site-packages\torch\nn\modules\module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "f:\codes\python apps\asr\speechbrain-released\speechbrain\nnet\containers.py", line 144, in forward
    x = layer(x)
  File "C:\Users\flozi\anaconda3\envs\wav2vec\lib\site-packages\torch\nn\modules\module.py", line 889, in _call_impl
    result = self.forward(*input, **kwargs)
  File "f:\codes\python apps\asr\speechbrain-released\speechbrain\nnet\CNN.py", line 394, in forward
    x = self._manage_padding(
  File "f:\codes\python apps\asr\speechbrain-released\speechbrain\nnet\CNN.py", line 446, in _manage_padding
    x = F.pad(x, padding, mode=self.padding_mode)
  File "C:\Users\flozi\anaconda3\envs\wav2vec\lib\site-packages\torch\nn\functional.py", line 4036, in _pad
    raise NotImplementedError("Only 3D, 4D, 5D padding with non-constant padding are supported for now")
NotImplementedError: Only 3D, 4D, 5D padding with non-constant padding are supported for now
```

**Comment by TParcollet** [comment]
Hey, you need to un squeeze the wav file to fake thé batch dimension

**Comment by flozi00** [comment]
ah okay
It's the code copied from here: https://huggingface.co/speechbrain/mtl-mimic-voicebank
could you change it there too ?

**Comment by TParcollet** [comment]
@pplantinga

**Comment by pplantinga** [comment]
Addressed in 5fbae27c
